### PR TITLE
docs(pie-docs): DSW-2220 add overview page and assets

### PR DESCRIPTION
**Pull request by leksaBoiko**
## Describe your changes (can list changeset entries if preferable)

**[Added] Checkbox overview page**

alternative text: [open](https://docs.google.com/spreadsheets/d/1c8g1U9QY1TkEZsk_E80uUUHv-MDOz8eSlRI-fMc-7Vs/edit?gid=1755118177#gid=1755118177)
image assets: [open](https://drive.google.com/drive/folders/1FooGdclwccqYFT296oQnOMz1dA9fKUZ_)

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the `PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving
- [x] If changes will affect consumers of the package, I have created a changeset entry.

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2
- [x] I have reviewed the `PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by ashleynolan — APPROVED** [review]

File: .changeset/afraid-chairs-develop.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Added] Checkbox overview page
```
Comment (ashleynolan) [inline .changeset/afraid-chairs-develop.md:5]: Super small one:

```suggestion
[Added] - Checkbox overview page
```

**Review by ashleynolan — DISMISSED** [review]

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+          "Ensure labels are concise and descriptive that clearly indicates what selecting it will entail."
```
[the remark this answers is not in the document]
Comment (xander-marjoram) [reply apps/pie-docs/src/components/checkbox/overview/overview.md:29]: Yeah that's incorrect, also unnecessarily complex wording 😄 
I think this is what it's trying to say:
```suggestion
          "Make sure labels are concise and clearly describe what checking the box means."
```

**Review by raoufswe — APPROVED** [review]

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+---
+
+## Content
+
+All content should use sentence case.
+
+### Label
+
+- Always use clear and concise labels for checkboxes.
+- Labels appear to the right of checkbox input.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox/overview/overview.md:115]: Thoughts?
```suggestion
- Labels appear to the right of checkbox input (when the text direction is left-to-right).
```

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Examples
+
+Outlines the atomic level interactive elements for the component.
+
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox/overview/overview.md:250]: I think I remember there was a discussion saying that this piece of text only belongs in the "Interactive states" section above. @tingting-lau?

```suggestion
```

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+### LTR examples
+
+Here are some examples of the component in left-to-right context:
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox/overview/overview.md:253]:
```suggestion
Here are some examples of the component in a left-to-right context:
```

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+      src: "../../../assets/img/components/checkbox/checkbox-LTR-agreement.svg",
+      width: "375px",
+      alt: "Checkbox labeled 'I have read and accepted the terms and conditions' in left-to-right context."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-checked-LTR.svg",
+      width: "360px",
+      alt: "Selected checkbox followed by the label text in left-to-right context."
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox/overview/overview.md:267]:
```suggestion
      alt: "Selected checkbox followed by the label text in a left-to-right context."
```

File: apps/pie-docs/src/components/checkbox/overview/overview.md [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+      src: "../../../assets/img/components/checkbox/checkbox-RTL-agreement.svg",
+      width: "271px",
+      alt: "Checkbox labeled 'I have read and accepted the terms and conditions' in right-to-left context."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-checked-RTL.svg",
+      width: "360px",
+      alt: "Selected checkbox followed by the label text in right-to-left context."
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox/overview/overview.md:288]:
```suggestion
      alt: "Selected checkbox followed by the label text in a right-to-left context."
```

File: apps/pie-docs/src/components/checkbox/overview/overview.md (hunk 4 above)
Comment (tingting-lau) [reply apps/pie-docs/src/components/checkbox/overview/overview.md:250]: That is correct :) Thanks Xander.

**Review by xander-marjoram — APPROVED** [review]

**Review by raoufswe — APPROVED** [review]

**Pull request merged**